### PR TITLE
(DPS-2749) Check management content

### PR DESCRIPTION
**Pull request by arturfracala**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/DPS-2749)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2749)

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: src/pages/management/configuration-api/index.mdx [hunk 1]
```diff
@@ -4075,4 +4075,4 @@ Informs that a user has seen events up to a specific time.
 
 # Contact us
 
-If you found a bug or a typo, you can let us know directly on GitHub. In case of any questions or feedback, don't hesitate to contact us at [developers@livechat.com] (mailto:developers@livechat.com). We'll be happy to hear from you!
+If you found a bug or a typo, you can let us know directly on GitHub. In case of any questions or feedback, don't hesitate to contact us at developers@livechat.com. We'll be happy to hear from you!
```
Comment (jakubsikora) [inline src/pages/management/configuration-api/index.mdx:4078]: Cant we just use `<a href="mailto:developers@livechat.com">developers@livechat.com</a>` ?

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 2]
```diff
@@ -4075,4 +4075,4 @@ Informs that a user has seen events up to a specific time.
 
 # Contact us
 
-If you found a bug or a typo, you can let us know directly on GitHub. In case of any questions or feedback, don't hesitate to contact us at [developers@livechat.com] (mailto:developers@livechat.com). We'll be happy to hear from you!
+If you found a bug or a typo, you can let us know directly on GitHub. In case of any questions or feedback, don't hesitate to contact us at developers@livechat.com. We'll be happy to hear from you!
```
Comment (jakubsikora) [inline src/pages/management/configuration-api/v3.4/index.mdx:4078]: Same here

**Review by jakubsikora — CHANGES_REQUESTED** [review]
Few suggestions

File: src/pages/management/limits/index.mdx [hunk 3]
```diff
@@ -26,7 +28,12 @@ Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing
 | Tracked visitors              | 400     | 400   | 400      | 400        |
 | Customers history             | 2 mos.  | 1 yr. | 2 yr.    | 4 yr.      |
 
-To find out more about resource limits and the possibilities to change them, <a href="#open-chat" onClick={openChatWindow}>contact our support</a>.
+To find out more about resource limits and the possibilities to change them, <button style={{ background: 'none!important',
```
[the remark this answers is not in the document]
Comment (arturfracala) [reply src/pages/management/limits/index.mdx:31]: Link wasn't working here, so I changed the link to the button and added this css to keep the look. This site is hidden anyway, so I wouldn't care too much about it

**Review by jakubsikora — APPROVED** [review]

**Pull request merged**